js/multilingual_survey_settings.js

```diff
@@ -550,22 +633,59 @@ Multilingual.loadSurveySettings = function() {
 			}
 		}
 	})
+	
+	// update hide_lang checkbox
+	if (collections.basic_settings.hide_lang == '1') {
+		$("#hide_lang").prop('checked', true)
+	} else {
+		$("#hide_lang").prop('checked', false)
+	}
+	
 }
 
 Multilingual.getSettings();
 
+Multilingual.translateSettings = function() {
+	// get selected language from #ml-translate-language
+	var language_iso_code = $("select#ml-translate-language").val();
+	if (!language_iso_code) {
+		return;
+	}
+	
+	$.ajax({
+		url: Multilingual.ajax_url + '&translate_settings_iso_code=' + language_iso_code,
```

Comment:
I assume this ajax call has been pretty well tested manually.

Reply:
Yes.